src/solace_agent_mesh/agent/utils/artifact_helpers.py

```diff
@@ -819,7 +820,7 @@ async def load_artifact_content_or_metadata(
 
     if max_content_length is None and component:
         max_content_length = component.get_config(
-            "text_artifact_content_max_length", 1000
+            "text_artifact_content_max_length", 100000
```

Comment:
You should check the 'get_config' call. It should always return a value - either the one that is set or the default value. Then we don't need this logic here - just raise a ValueError if it is None